tests/foreman/sys/test_pulp3_filesystem.py

```diff
@@ -96,3 +99,28 @@ def test_pulp_workers_status(target_sat):
     output = target_sat.execute('systemctl status pulpcore-worker* | grep Active')
     result = output.stdout.rstrip().splitlines()
     assert all('Active: active (running)' in r for r in result)
+
+
+@pytest.mark.tier1
+def test_pulp_status(target_sat):
+    """Test pulp status via pulp-cli.
+
+    :id: aa7fda5a-cfa1-4fa1-8714-8d5a2e2f89d9
+
```

Comment:
Could you add the `:steps:` section to the docstring?

Reply:
Added.